.dir-locals.el

```diff
@@ -0,0 +1,4 @@
+((nil . ((indent-tabs-mode . t)
```

Comment:
We usually don't include editor preferences into the main repo. Think of all the Vim users who'd be shocked by the mere hint of an ELisp file on their system!